mysql-test/suite/rpl/t/rpl_percona_userstat.test

```diff
@@ -0,0 +1,60 @@
+--echo #
+--echo # Bug 995624 RBR events are not reflected in userstat's
+--echo # Rows_updated
+--echo #
+
```

Comment:
As discussed on Slack. We want the same output for mix, row and statement. This would limit the output just to mixed and row.